besu/src/main/java/org/hyperledger/besu/cli/options/TransactionPoolOptions.java

```diff
@@ -122,6 +123,15 @@ public class TransactionPoolOptions implements CLIOptions<TransactionPoolConfigu
       arity = "1..*")
   private Set<Address> prioritySenders = TransactionPoolConfiguration.DEFAULT_PRIORITY_SENDERS;
 
+  @CommandLine.Option(
+      names = {TX_POOL_MIN_GAS_PRICE},
+      paramLabel = "<Wei>",
+      description =
+          "Minimum price (in Wei) offered by a transaction to be added to the txpool (not to be confused with min-gas-price,"
+              + " that is applied on block creation) (default: ${DEFAULT-VALUE})",
```

Comment:
Transactions with gas price lower than this minimum will not be accepted into the txpool

Reply:
updated the description